src/apps/WalletConnect/hooks/useWalletConnect.tsx

```diff
@@ -53,22 +53,35 @@ const useWalletConnect = () => {
           throw error;
         }
 
-        if (payload.method === "eth_sendTransaction") {
-          const txInfo = payload.params[0];
-          safe.sendTransactions([
-            {
-              to: txInfo.to,
-              value: txInfo.value || "0x0",
-              data: txInfo.data || "0x",
-            },
-          ]);
-        } else {
-          wcConnector.rejectRequest({
-            id: payload.id,
-            error: {
-              message: "METHOD_NOT_SUPPORTED",
-            },
-          });
+        switch (payload.method) {
+          case "eth_sendTransaction": {
+            const txInfo = payload.params[0];
+            safe.sendTransactions([
+              {
+                to: txInfo.to,
+                value: txInfo.value || "0x0",
+                data: txInfo.data || "0x",
+              },
+            ]);
+            break;
+          }
+          case "gs_multi_send": {
+            const txs = payload.params as any[];
```

Comment:
🙈

Reply:
We should add this type in some generic place. 
```
interface Transaction {
    to: string;
    value: string;
    data: string;
}
```

Reply:
It can be imported from the sdk

Reply:
hmm the definition of a JsonRPC is that it is `any[]`, that's why I chose it. But we could use the Transaction type here as we expect the 3 values to be present.

Reply:
That is also why I added the todo, since we do blind casting from `any[]` to `Transaction[]` ... not a big fan of that

Reply:
I understand your concern now and why we may want to perform validation there.
Could it be that `unknown` is a better option? We can also make a type-guard to check if it has a structure similar to the transaction object: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

Reply:
As this looks good I will check this out on friday :)